rmgpy/solver/simpleTest.py

```diff
@@ -44,11 +44,50 @@
 import rmgpy.constants as constants
 from rmgpy.chemkin import loadChemkinFile
 from rmgpy.rmg.settings import ModelSettings, SimulatorSettings
+from rmgpy.data.rmg import RMGDatabase
+from rmgpy.rmg.main import RMG
+from rmgpy import settings
+from rmgpy.tools.loader import loadRMGPyJob
 
 ################################################################################
+class ConcentrationPrinter(object):
```

Comment:
Same style changes here (two lines above and after, and extra space before comma)